src/realmGeolBasin.ttl

```diff
@@ -96,25 +101,49 @@ soreagb:PeripheralForelandBasin rdf:type owl:Class ;
 
 ###  http://sweetontology.net/realmGeolBasin/RetroarcForelandBasin
 soreagb:RetroarcForelandBasin rdf:type owl:Class ;
-                            rdfs:subClassOf soreagb:Basin ;
+                            rdfs:subClassOf soreagb:SedimentaryBasin ;
                             rdfs:label "retroarc foreland basin"@en .
 
 
+###  http://sweetontology.net/realmGeolBasin/Subbasin
+soreagb:Subbasin rdf:type owl:Class ;
+                     dcterms:contributor <http://linked.data.gov.au/org/gsq> ;
+                     rdfs:subClassOf soreagb:SedimentaryBasin ;
+                     skos:altLabel "sub-basin"@en ;
+                     skos:definition "A sedimentary basin may be subdivided into two or more subbasins that have separate depocentres and are separated from one another by some tectonic/structural element, such as a fault or basement high. (Definition created here: Paul Blake, John McKellar)"@en ;
+                     skos:prefLabel "subbasin"@en .
+
+
 ###  http://sweetontology.net/realmGeolBasin/SuccessorBasin
 soreagb:SuccessorBasin rdf:type owl:Class ;
-                     rdfs:subClassOf soreagb:Basin ;
+                     rdfs:subClassOf soreagb:SedimentaryBasin ;
                      rdfs:label "successor basin"@en .
 
 
+###  http://sweetontology.net/realmGeolBasin/Superbasin
+soreagb:Superbasin rdf:type owl:Class ;
+                     dcterms:contributor <http://linked.data.gov.au/org/gsq> ;
+                     rdfs:subClassOf soreagb:SedimentaryBasin ;
+                     skos:definition "A superbasin is a hierarchial term applied to  a group of largely interconnected or associated sedimentary basins of the same or similar age. Its component basins may be separated or partially separated by tectonic elements that define 'boundaries'  between them, or they may have been subjected to somewhat different geological influences. In the Australian Mesozoic, the Great Australian Superbasin, composed of the Nambour, Clarence-Moreton, Surat, Eromanga, Carpentaria and other basins, is a prime example, variously covering a geographically-extensive area in parts of Queensland, the Northern Territory, New South Wales and South Australia. (Definition created here)"@en ;
```

Comment:
define the concept. Currently what it says is 'a superbasin is a term'. I don' think that's what we want. 
suggest def:
A sedimentary basin that includes several interconnected sedimentary basins of the same or similar age.

question-- is being interconnected a necessary/sufficient property of parts of a superbasin. That's the implication of the proposed definition.   
2.  a basin hasPart subbasin; also a superbasin hasPart basin.  What differentiates soreagb:SedimentaryBasin with sub-basins from a soreagb:Superbasin that includes basins?

ScopeNote-- rest of the stuff in the current definition